src/components/Facility/FacilityHome.tsx

```
@@ -145,7 +145,15 @@ export const FacilityHome = ({ facilityId }: Props) => {
       queryClient.invalidateQueries({
         queryKey: ["facility", facilityId],
       });
-      navigate("/");
+
+      const prevPath = document.referrer;
+      const orgMatch = prevPath.match(/\/org\/([^/]+)\/facilities/);
+
+      if (orgMatch && orgMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)